wmibeat.yml

```diff
@@ -206,7 +215,7 @@ output:
 
 
   ### Console output
-  # console:
+  console:
```

Comment:
Is this a deliberate change, or was this something you did for debugging to check values in the console?

Reply:
Yea I should have made this a separate, clearer commit. Beats require at least one output to run. Previously the default file had an elastic instance, but that means you get errors of it failing to connect to elastic so I thought it would be more useful to have it output to the console by default